docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -36,18 +40,18 @@ on existing checks.
 
 **Examples**:
 
-- When the `RECORD_DEF` token was introduced, it was reasonable to look at all checks
-  that had `CLASS_DEF` in their acceptable tokens.
+- **Check updates**: When the `RECORD_DEF` token was introduced,
+ it was reasonable to look at all checks that had `CLASS_DEF` in their acceptable tokens.
 
-- When the `TEXT_BLOCK_CONTENT` token was introduced, it was reasonable to look at all checks
-  that had `STRING_LITERAL` in their acceptable tokens.
+- **Check updates**: When the `TEXT_BLOCK_CONTENT` token was introduced,
+  it was reasonable to look at all checks that had `STRING_LITERAL` in their acceptable tokens.
 
-- When switch expressions were introduced, it was reasonable to look at all checks
-  that deals with expressions because `LITERAL_SWITCH` may now fall under `EXPR` token.
+- **Check updates**: When switch expressions were introduced, it was reasonable to look at all
+  checks that deals with expressions because `LITERAL_SWITCH` may now fall under `EXPR` token.
   This requires analyzing the impact of this new child token on existing checks,
   such as `InnerAssignmentCheck`
 
-### Frequently Impacted Checks
+## Frequently Impacted Checks
```

Comment:
Let's have a very short explanation for each of these checks

Reply:
We line to the checks documentation here, isn't this a redundant explanation? You can see a short description if you click the link and also these check names explain themselves in a good way. what do you think?

Reply:
I mean like why these should always be considered. Help to get the reader in the right frame of mind

Reply:
done